### PR TITLE
Add Corvallis file

### PR DESCRIPTION
**Pull request by doubwoods3**
This is the beginning of the Corvallis file. 🌞 

Please provide feedback on:

- If you've been to Corvallis, what did you like? What did you not like?
- If you've never been to Corvallis, what interests you about coming here?
- Are you interested in learning about lodging? What do you prefer? Hotels? AirBNBs? etc.
- @albeanth what kinds of things would your son like to do?
- @harterj do you know of any good 🍻  breweries 🍻  in or around Corvallis?

### CODE REVIEW

**Comment by harterj** [comment]
> @albeanth what kinds of things would your son like to do?


haha Tony with a son...disc golf! pickle ball!

**Comment by albeanth** [comment]
#**I have a kid??**

**Comment by albeanth** [comment]
# _I HAVE A KID??_

there we go haha

File: doubwoods3-Corvallis.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Corvallis, OR
+
+### Restaurants
+
+### Local Attractions
+
+- Oregon State University
+- OSTR
```
Comment (harterj) [inline doubwoods3-Corvallis.md]: I feel like you're trying to say "ostrich" here

**Comment by benisntfunny** [comment]
I like Oregon. **It has good Wine 🍷 !**

**Comment by albeanth** [comment]
@benisntfunny and beer!! 🍺 🍺

**Comment by albeanth** [comment]
@doubwoods3 is the TRIGA really an __attraction?__ 😆

**Comment by doubwoods3** [comment]
> I like Oregon. It has good Wine 🍷 !

**Comment by doubwoods3** [comment]
> I like Oregon. It has good Wine 🍷 !

It _does_ have good wine!

**Comment by doubwoods3** [comment]
> @doubwoods3 is the TRIGA really an attraction? 😆

Aren't you attracted to it?

File: doubwoods3-Corvallis.md (hunk 1 above)
Comment (doubwoods3) [inline doubwoods3-Corvallis.md]: but you have to yell it: "OSTRICH"

**Comment by doubwoods3** [comment]
@harterj @albeanth Thanks for the idears 💡  for ff9fda2

**Comment by doubwoods3** [comment]
:shipit:

**Comment by readiu** [comment]
Ready to :shipit:

**Comment by AllstonS** [comment]
Looks 👍   🚢

**Comment by doubwoods3** [comment]
> Looks 👍 🚢

Thanks!

**Pull request merged**